### PR TITLE
Add Ref to paragraph about updating data

### PR DESCRIPTION
**Pull request by ionas**

### CODE REVIEW

File: en/orm/query-builder.rst [hunk 1]
```diff
@@ -1110,6 +1110,7 @@ queries::
         ->values($select)
         ->execute();
 
+:ref:query-builder-updating-data
```
Comment (antograssiot) [inline en/orm/query-builder.rst]: .. _query-builder-updating-data:

**Comment by antograssiot** [comment]
I don't know how to explain it correctly and I'm sorry for that so if you have some time, please have a look at http://book.cakephp.org/3.0/en/contributing/documentation.html#cross-referencing-links
It will be clearer

**Comment by antograssiot** [comment]
@ions are you willing to update this or should we fix it post merge ?

**Comment by ionas** [comment]
Sorry, lots of deadlines and pressure, especially in regards to twitters' typeahead lib (meh)... I hope It is good to merge now.

**Comment by antograssiot** [comment]
No worries @ionas  thanks for helping us :+1:

**Comment by ionas** [comment]
Nah, its more like you are helping me. I will try to contribute more next year - if possible :).

**Pull request merged**